Installation/Schema/klant.betrokkeneBijKlantcontact.schema.json

```diff
@@ -2,56 +2,61 @@
-  "required": ["klantcontactId", "rol"],
+  "required": ["klantcontact", "rol"],
   "properties": {
-    "bezoekadres": {
+    "klantcontact": {
       "type": "object",
-      "description": "Een bezoekadres is een adres waar een persoon of een bedrijf bezoekers ontvangt.",
-      "$ref": "https://commongateway.nl/klant.bezoekadres.schema.json"
+      "$ref": "https://commongateway.nl/klant.klantcontact.schema.json"
     },
-    "correspondentieadres": {
+    "partij": {
       "type": "object",
-      "description": "Het adres waarnaar de persoon of de organisatie de voor hen bestemde correspondentie laat verzenden.",
-      "$ref": "https://commongateway.nl/klant.correspondentieadres.schema.json"
+      "$ref": "https://commongateway.nl/klant.partij.schema.json"
     },
-    "klantcontactId": {
+    "rol": {
       "type": "string",
+      "description": "De typering van de rol van de persoon of organisatie in het klantcontact.",
       "required": true,
-      "maxLength": 40
+      "enum": [
+        "vertegenwoordiger",
+        "klant"
+      ]
+    },
+    "intitiator": {
+      "type": "boolean",
+      "required": true,
+      "default": "false"
     },
     "organisatienaam": {
       "type": "string",
       "maxLength": 200
     },
-    "partijId": {
-      "type": "string",
-      "maxLength": 40
-    },
-    "persoonsnaam": {
+    "contactnaam": {
       "type": "object",
       "$ref": "https://commongateway.nl/klant.persoonsnaam.schema.json"
     },
-    "rol": {
-      "type": "string",
-      "description": "De typering van de rol van de persoon of organisatie in het klantcontact.",
-      "required": true,
-      "enum": [
-        "vertegenwoordiger",
-        "klant"
-      ]
+    "bezoekadres": {
+      "type": "object",
+      "description": "Een bezoekadres is een adres waar een persoon of een bedrijf bezoekers ontvangt.",
+      "$ref": "https://commongateway.nl/klant.bezoekadres.schema.json"
+    },
+    "correspondentieadres": {
+      "type": "object",
+      "description": "Het adres waarnaar de persoon of de organisatie de voor hen bestemde correspondentie laat verzenden.",
+      "$ref": "https://commongateway.nl/klant.correspondentieadres.schema.json"
     },
-    "voorkeurskanaal": {
+    "communicatiekanaal": {
       "type": "object",
       "description": "Het kanaal waarlangs de partij bij voorkeur interactie heeft met de gemeente.",
-      "$ref": "https://commongateway.nl/klant.voorkeur.schema.json"
+      "$ref": "https://commongateway.nl/klant.digitaalAdres.schema.json"
```

Comment:
Een adres voor communicatiekanaal voelt raar 🤔 
Dit is correct?

Reply:
![image](https://github.com/CommonGateway/CustomerInteractionBundle/assets/10433082/35efd5d0-3177-4eb5-a2e3-775fa79c4426)
Jup, vermoed dat hier het e-mail adres, telefoonnummer, faxnummer of wat dan ook in terecht komt ;)

Reply:
't leuke is hier vooral dat de termen communicatiekanaal en digitaal adres door elkaar heen gebruikt worden/werden. Ik kan ook de property hernoemen natuurlijk.